pkgs/misc/vim-plugins/overrides.nix

```diff
@@ -287,8 +287,11 @@ self: super: {
   });
 
   sql-nvim = super.sql-nvim.overrideAttrs(old: {
-    buildInputs = [ sqlite ];
-  });
+    preFixup = ''
```

Comment:
```suggestion
    postPatch = ''
```

Reply:
@SuperSandro2000 thanks for reviewing. I don't mind to do the change, the output will the same. But can you explain the reason for it? I thought that we should use the fixup phase for nix-specific post-processing actions, like in this case, setting the correct path to a .so file in the nix store.

Reply:
fixup is usually used for fixing rpaths in binaries or similar. It does not make any difference here but code changes are usually done in patch.

Reply:
@SuperSandro2000 thanks for clarifying it, I just updated it, I'll keep that in mind for future derivations